nixos/tests/all-tests.nix

```diff
@@ -889,6 +889,7 @@ in {
   rathole = handleTest ./rathole.nix {};
   readarr = handleTest ./readarr.nix {};
   realm = handleTest ./realm.nix {};
+  readeck = handleTest ./readeck.nix {};
```

Comment:
Please use runTest

Reply:
I am not sure what is the difference between `handleTest` and `runTest`, but when I change to `runTest` I get an infinite recursion.

Reply:
You need to update your test file too. Just check how other tests using `runTest` are written, you'll find the solution pretty quickly.